subscribe/lambda/function.rb

```diff
@@ -90,12 +93,22 @@ def cloudwatch_log_group_name
         "/aws/lambda/#{name}"
       end
 
+      def subscriptions
+        @subscriptions ||= begin
```

Comment:
does this will work in the case that you have no subscriptions yet? when you are spinning it for the first time?

Reply:
Yes it does. 

The AWS SDK returns an object instance with a `subscription_filters` method that returns an empty array when there are no subscriptions present.

We memoize the response as it's referenced a few times to avoid unnecessary API calls.